Lombiq.Hosting.Tenants.EmailQuotaManagement.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -11,7 +13,8 @@ public static class TestCaseUITestContextExtensions
 {
     private const string SuccessfulSubject = "Successful test message";
     private const string UnSuccessfulSubject = "Unsuccessful test message";
-    private const string DashboardWarning =
+    private const string WarningSubject = "[Warning] Your DotNest site has used";
```

Comment:
You are right, the subjects should be generic in the module. But I'm not sure how could we override that from another module other than using localization files, but I don't think we want that.
Should I then create a service that gives back the subject and then others can override that from their module with their own service? Or any suggestions because I'm really stuck with this? Or it could be generic always.

Reply:
A service to return these two subjects (as localized strings) would be good, yes. It looks like an overkill but I can't think of a better way either (site settings are not suitable for a SaaS and are bad for localization, options are bad for localization, rendering shapes for just these strings would make overriding perhaps easier but would be an even bigger overkill).